styleguide.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 	components: 'src/components/*/*.vue',
 	getComponentPathLine(componentPath) {
 		const name = path.basename(componentPath, '.vue')
-		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}'`
```

Comment:
so are those parentheses not needed anymore or why are we removing them from the docs?

Reply:
No they're actually breaking.
We don't export submodules when getting the direct path.
See the readme: https://github.com/nextcloud/nextcloud-vue#usage

> To use a component, just import it:
> `import { AppNavigation } from '@nextcloud/vue'`
> 
> Depending on which components you use, you might want to only import individual (separately bundled) components:
> `import Avatar from '@nextcloud/vue/dist/Components/Avatar'`

Reply:
thanks for the explanation @skjnldsv !